docs/faq.rst

```diff
@@ -300,6 +292,43 @@ Common reasons:
   A quick (though not precise) way to check it is to try opening a page
   in Safari.
 
+**Splash crashes.**
+
+Common reasons:
+
+* Qt or WebKit bugs which cause Splash to hang or crash. Often the whole
+  website works, but some specific .js (or other) file causes problems.
+  In this case you can try this steps:
+  
+  * Run Splash locally with v2 verbosity, e.g. ``docker run -it -p8050:8050 scrapinghub/splash -v2``
+  * Go to ``http://0.0.0.0:8050`` and paste your url (with default lua)
+  * Look at the log in terminal.
+  * If Splash instance failed and stopped - go to the next steps. For example the last request was to the url like ``https://.../static/tagtag.min.js`` with JS,  and we can suspect that probably it caused the issue.
+  * Try to filter out this script with using :ref:`splash-on-request` with this template (or use use :ref:`request filters` to filter it out) :
+  
+    .. code-block:: lua
+
+     function main(splash, args)
+     splash:on_request(function(request)
+       if request.url:find('tagtag') ~= nil then
+             request:abort()
+         end
```

Comment:
it seems indentation is a bit off

Reply:
it'd be also good to use consistent indentation - here 4 spaces are used, while in return statement 2 spaces are used